package.json

```diff
@@ -4,7 +4,9 @@
   "description": "PrototypeAndela",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "log:value": "node scripts/logging-values",
```

Comment:
This looks good. 

Hope you ran `npm run log:value` in your terminal?

Reply:
Yes i ran both on the terminal. it works!

Reply:
Okay. Effect the other changes then.